alerting/config.go

```diff
@@ -1,6 +1,7 @@
 package alerting
 
 import (
+	"github.com/TwiN/gatus/v5/alerting/provider/zulip"
 	"log"
 	"reflect"
 	"strings"
```

Comment:
Can you move the `"github.com/TwiN/gatus/v5/alerting/provider/zulip"` import to the 2nd section with the other non-standard imports?